flink-core/src/test/java/org/apache/flink/api/java/typeutils/PojoParametrizedTypeExtractionTest.java

```diff
@@ -0,0 +1,101 @@
+package org.apache.flink.api.java.typeutils;
+
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.Serializable;
+
+/**
+ *  Tests concerning type extraction of Parametrized Pojo and its superclasses.
+ */
+public class PojoParametrizedTypeExtractionTest {
+
+	@Test
+	public void testCreateTypeInfo(){
+		/// Init
+		final TypeInformation<ParametrizedParentImpl> directTypeInfo = TypeExtractor.createTypeInfo(ParametrizedParentImpl.class);
+
+		/// Check
+		Assert.assertTrue(directTypeInfo instanceof PojoTypeInfo);
+	}
+
+	@Test
+	public void testFieldExtraction(){
```

Comment:
I tend to merge this PR with the above `testCreateTypeInfo` and rename to `testDirectlyCreateTypeInfo`, since here seems to have the same action with the above test and with more asserts. This makes them more like the same test.

If we merge the two tests, we may create an explicit `PojoTypeInfo` object as the expected result and use the `assertEquals` to compare the result with the expected result.